### PR TITLE
Fix used memory calculations in docker/memory

### PR DESCRIPTION
**Pull request by fearful-symmetry**
## What does this PR do?

This brings the memory usage reporting for `docker/memory` in line with `docker stat`, as well as `cadvisor` and `containerd` by calculating used memory as `used - inactive_file`. I'm marking this a bug, since we were the only ones using this more simplistic usage calculation.

## Why is it important?

This brings us in line with how container memory usage is calculated elsewhere.

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## How to test this PR locally

- Pull down and build
- Run on a system with docker and containers running
- Make sure the value for container memory `usage` in `docker/memory` is a reasonable value.

## Related issues

- closes https://github.com/elastic/beats/issues/26751

### CODE REVIEW

File: metricbeat/module/docker/memory/_meta/data.json [hunk 1]
```diff
@@ -1,61 +1,94 @@
...
-                "writeback": 0
+                "workingset_activate": 0,
+                "workingset_nodereclaim": 0,
+                "workingset_refault": 0
             },
             "usage": {
-                "max": 7860224,
-                "pct": 0.000672283359618831,
-                "total": 1409024
+                "max": 0,
```
Comment (kaiyan-sheng) [inline metricbeat/module/docker/memory/_meta/data.json:89]: Does this 0 look right to you?

Comment (fearful-symmetry) [reply metricbeat/module/docker/memory/_meta/data.json:89]: This is a case where V1 and V2 might be different. `Max` Just indicates the cgroup's upper threshold, with `0` meaning there is no upper limit.

**Review by jsoriano — APPROVED** [review]
:+1:

**Comment by jsoriano** [comment]
> I'm marking this a bug

Do you want to backport this to 7.15? :slightly_smiling_face:

**Comment by fearful-symmetry** [comment]
@jsoriano Added the correct (I think) backport tags.

**Pull request merged**